data.sql

```diff
@@ -1,19 +1,97 @@
-INSERT INTO animals ("name", "date_of_birth", "escape_attempts", "neutered", "weight_kg") VALUES ('Boarmon', '2005-06-07', 7, 't', 20.4);
-INSERT INTO animals ("name", "date_of_birth", "escape_attempts", "neutered", "weight_kg") VALUES ('Blossom', '1998-10-13', 3, 't', 17);
-INSERT INTO animals ("name", "date_of_birth", "escape_attempts", "neutered", "weight_kg") VALUES ('Ditto', '2022-05-14', 4, 't', 22);
-
-
-INSERT INTO animals ("id", "name", "date_of_birth", "escape_attempts", "neutered", "weight_kg", "species") VALUES
-(1, 'Charmander', '2020-02-08', 0, 'f', -11, NULL),
-(2, 'Plantmon', '2021-11-15', 2, 't', -5.7, NULL),
-(3, 'Squirtle', '1993-04-02', 3, 'f', -12.13, NULL),
-(4, 'Angemon', '2005-06-12', 1, 't', -45, NULL),
-(5, 'Boarmon', '2005-06-07', 7, 't', 20.4, NULL),
-(6, 'Blossom', '1998-10-13', 3, 't', 17, NULL),
-(7, 'Ditto', '2022-05-14', 4, 't', 22, NULL);
+INSERT INTO vets ("name", "age", "date_of_graduation") VALUES ('Vet William Tatcher', 45, '2000-04-23');
+INSERT INTO vets ("name", "age", "date_of_graduation") VALUES ('Vet Maisy Smith', 26, '2019-01-17');
+INSERT INTO vets ("name", "age", "date_of_graduation") VALUES ('Vet Stephanie Mendez', 64, '1981-05-04');
+INSERT INTO vets ("name", "age", "date_of_graduation") VALUES ('Vet Jack Harkness', 38, '2008-06-8');
+
+
+
+-- Insert the following data for specializations: 
+
+
+INSERT INTO specializations ("vet_id", "species_id")
+		VALUES(( SELECT DISTINCT
+				id FROM vets
+			WHERE
+				name = 'William Tatcher'), ( SELECT DISTINCT
+					id FROM species
+				WHERE
+					name = 'Pokemon'));
+
+INSERT INTO specializations ("vet_id", "species_id")
+		VALUES(( SELECT DISTINCT
+				id FROM vets
+			WHERE
+				name = 'Stephanie Mendez'), ( SELECT DISTINCT
+					id FROM species
+				WHERE
+					name = 'Pokemon'));
+
+INSERT INTO specializations ("vet_id", "species_id")
+		VALUES(( SELECT DISTINCT
+				id FROM vets
+			WHERE
+				name = 'Stephanie Mendez'), ( SELECT DISTINCT
+					id FROM species
+				WHERE
+					name = 'Digimon'));
+
+INSERT INTO specializations ("vet_id", "species_id")
+		VALUES(( SELECT DISTINCT
+				id FROM vets
+			WHERE
+				name = 'Jack Harkness'), ( SELECT DISTINCT
+					id FROM species
+				WHERE
+					name = 'Digimon'));
+
+
+
+-- nsert the following data for visits: 
+
+INSERT INTO visits ("vet_id", "animal_id", "date_of_visit")
+		VALUES(( SELECT DISTINCT
+				id FROM vets
+			WHERE
+				name = 'William Tatcher'), ( SELECT DISTINCT
+					id FROM animals
+				WHERE
+					name = 'Agumon'),
+					'2020-05-24'
+					);
+
+
+INSERT INTO visits ("vet_id", "animal_id", "date_of_visit")
+		VALUES(( SELECT DISTINCT
+				id FROM vets
+			WHERE
+				name = 'Stephanie Mendez'), ( SELECT DISTINCT
+					id FROM animals
+				WHERE
+					name = 'Agumon'),
+					'2020-07-22'
+					);
+
+INSERT INTO visits ("vet_id", "animal_id", "date_of_visit")
+		VALUES(( SELECT DISTINCT
+				id FROM vets
```

Comment:
- [ ] In order for the queries to work as expected please make sure to insert all the required data for visits:

    Agumon visited William Tatcher on May 24th, 2020.
    Agumon visited Stephanie Mendez on Jul 22th, 2020.
    Gabumon visited Jack Harkness on Feb 2nd, 2021.
    Pikachu visited Maisy Smith on Jan 5th, 2020.
    Pikachu visited Maisy Smith on Mar 8th, 2020.
    Pikachu visited Maisy Smith on May 14th, 2020.
    Devimon visited Stephanie Mendez on May 4th, 2021.
    Charmander visited Jack Harkness on Feb 24th, 2021.
    Plantmon visited Maisy Smith on Dec 21st, 2019.
    Plantmon visited William Tatcher on Aug 10th, 2020.
    Plantmon visited Maisy Smith on Apr 7th, 2021.
    Squirtle visited Stephanie Mendez on Sep 29th, 2019.
    Angemon visited Jack Harkness on Oct 3rd, 2020.
    Angemon visited Jack Harkness on Nov 4th, 2020.
    Boarmon visited Maisy Smith on Jan 24th, 2019.
    Boarmon visited Maisy Smith on May 15th, 2019.
    Boarmon visited Maisy Smith on Feb 27th, 2020.
    Boarmon visited Maisy Smith on Aug 3rd, 2020.
    Blossom visited Stephanie Mendez on May 24th, 2020.
    Blossom visited William Tatcher on Jan 11th, 2021.